dpnp/dpnp_iface_statistics.py

```diff
@@ -668,8 +668,8 @@ def median(a, axis=None, out=None, overwrite_input=False, keepdims=False):
     overwrite_input : bool, optional
        If ``True``, then allow use of memory of input array `a` for
        calculations. The input array will be modified by the call to
-       ``median``. This will save memory when you do not need to preserve
-       the contents of the input array. Treat the input as undefined,
+      :obj:`dpnp.median`. This will save memory when you do not need to
```

Comment:
a space is missing in indention
```suggestion
       :obj:`dpnp.median`. This will save memory when you do not need to
```